### ISSUE TITLE
ViNCEd

### ISSUE THREAD

**Issue by polluks** [sodero/InstallerLG#68]
Something went wrong
````
[4152:makeassign] Weise '..VNC..' an '/' zu
[4161:run] '"SYS:Utilities/Multiview" ..VNC..:ReadMe.First!' wurde ausgeführt.
[4206:copyfiles] Kopiert '..VNC..:LIBS/vnc.library' nach 'SYS:LIBS/vnc.library'.
[4263:copyfiles] Kopiert '..VNC..:C/SetVNC.coded' nach 'T:SetVNC.coded'.
[4265:run] '..VNC..:Extras/Decode T:SetVNC.coded' wurde ausgeführt.
[4269:run] 'Wait 2' wurde ausgeführt.
[4271:rename] Umbenannt 'T:SetVNC.coded' in 'T:SetVNC'.
[4275:copyfiles] Kopiert 'T:SetVNC' nach 'SYS:C/SetVNC'.
[4277:delete] 'T:SetVNC' gelöscht.
[87:copyfiles] Kopiert '..VNC..:LOCALE/catalogs/Deutsch/VNC/VNC.prefs.catalog' nach 'SYS:Locale/Catalogs/Deutsch/VNC/VNC.prefs.catalog'.
[87:copyfiles] Kopiert '..VNC..:LOCALE/catalogs/Deutsch/VNC/VNC.sys.catalog' nach 'SYS:Locale/Catalogs/Deutsch/VNC/VNC.sys.catalog'.
[87:copyfiles] Kopiert '..VNC..:LOCALE/catalogs/Français/VNC/VNC.prefs.catalog' nach 'SYS:Locale/Catalogs/Français/VNC/VNC.prefs.catalog'.
[13:makeassign] Entferne Zuweisung '..VNC..'
````
http://aminet.net/util/shell/ViNCEd.lha

**Comment by sodero** [comment]
Thanks for reporting this. I don't understand the problem by reading the log above though. What's going wrong?

**Comment by polluks** [comment]
![VNC](https://user-images.githubusercontent.com/4707573/120761039-4d28e980-c504-11eb-97bc-64050d9e6911.png)

**Comment by sodero** [comment]
Thanks. I'll do some digging during the weekend.

**Comment by sodero** [comment]
Ok, more digging necessary than hours left this weekend.

**Comment by polluks** [comment]
I see. However the copied files seem to change with every start...

2021-06-06 21:07 GMT+02:00, Ola Söder ***@***.***>:

**Comment by sodero** [comment]
Sigh. Major setback, my MorphOS machine broke down when looking at this. I'll resume once I have a new PSU.

**Comment by polluks** [comment]
Still broken...
![VNC](https://user-images.githubusercontent.com/4707573/131626573-37a7f3e8-f96e-40d0-a626-d535b972e5e8.png)

**Comment by sodero** [comment]
Strange. On my machine it works and behaves exactly like the CBM installer. I'm using "intermediate" user and use all the default choices, yes, yes, yes and so on. What choices do you make?

**Comment by polluks** [comment]
Indeed, use "expert".

**Comment by sodero** [comment]
Excellent, I'll try again tonight.

**Comment by sodero** [comment]
Fixed in 5c7940684620163a702160e95acdabc77e9b6246
